### PR TITLE
feat/typescript

### PR DESCRIPTION
**Pull request by DenislavK8**

### CODE REVIEW

File: src/Form.tsx [hunk 1]
```diff
@@ -1,10 +1,26 @@
-import React, { useState } from "react";
+import { useState } from "react";
 import "./Form.css";
 
-const Form = ({ onFormSubmit, games, languages }) => {
-  const [visible, setVisible] = useState(false);
-  const [campaign, setCampaign] = useState({
-    name: "",
+interface Campaign {
```
Comment (preslavsh) [inline src/Form.tsx:4]: You have one here and one src/campaign.ts which is right one?

File: src/campaign.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export interface Campaign {
```
Comment (preslavsh) [inline src/campaign.ts:1]: store your interfaces in src/interfaces/yourfile

File: src/index.tsx [hunk 3]
```diff
@@ -14,4 +14,4 @@ root.render(
 // If you want to start measuring performance in your app, pass a function
 // to log results (for example: reportWebVitals(console.log))
 // or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+reportWebVitals(console.log);
```
Comment (preslavsh) [inline src/index.tsx:17]: why we have changes in this file?

File: src/Form.tsx (hunk 1 above)
Comment (DenislavK8) [reply src/Form.tsx:4]: Okey, now I import this from src/interfaces/campaign.ts in Form.tsx.

File: src/campaign.ts (hunk 2 above)
Comment (DenislavK8) [reply src/campaign.ts:1]: done

File: src/Form.tsx [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
+  name: string;
+  description: string;
+  game: string;
+  start: string;
+  end: string;
+  budget: string;
+  language: string;
+}
+
+interface Props {
```
[the remark this answers is not in the document]
Comment (DenislavK8) [reply src/Form.tsx:14]: done, also in Table.tsx

**Pull request merged**